src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -388,6 +392,21 @@ public IActionResult LaunchDebugger()
         [Authorize(Policy = PolicyNames.AdminAuthLevelOrInternal)]
         public async Task<IActionResult> SyncTriggers()
         {
+            _metricsLogger.LogEvent(MetricEventNames.SyncTriggersInvoked);
```

Comment:
These two new metrics will allow us to calculate the percentage of externally invoked sync operations that are made when the host is in a running state.